Add deploy readme and users-chis-{ke,ug} values (#1)
medic/cht-user-management: scripts/deploy/medic-deploy.md

> ## Medic Deployment
> 

Reviewer: We should add an intro here for folks who have never used these tools or it's been a long time since they have.  Suggesting something, but it's just a suggestion!

```suggestion
## Medic Deployment

This readme talks about how to use `helm` and `kubect`  running locally on your workstation to create, update and delete instances of the CHT User Management tool.  These will be running in Medic's [EKS](https://docs.aws.amazon.com/eks/latest/userguide/what-is-eks.html) and these instructions are meant to be followed by Medic teammates who have access to EKS.

General public is welcome to look at these instructions for who they might use them in their own infrastructure. 

### Key/Value pairs used 
```

| Key       | Value                                                   |
|-----------|---------------------------------------------------------|
| context   | arn:aws:eks:eu-west-2:720541322708:cluster/prod-cht-eks |
| namespace | users-chis-prod                                         |

### Requirements:

> 1. Applications:
>   - [Helm](https://helm.sh/)
>   - [Kubectl](https://kubernetes.io/docs/reference/kubectl/kubectl/)
> 
> 2. [Add medic helm chart repository](https://github.com/medic/helm-charts/tree/main#usage)
> 
> 3. [Authenticate to Medic kubernetes cluster (EKS)](https://github.com/medic/medic-infrastructure/blob/master/terraform/aws/dev/eks/access/README.md)

Reviewer: ```suggestion
-  Have both [Helm](https://helm.sh/) and  [Kubectl](https://kubernetes.io/docs/reference/kubectl/kubectl/) installed on your local workstation
- Check out [helm chart repository](https://github.com/medic/helm-charts/tree/main#usage) so you can reference it locally
- Be able to [authenticate to Medic kubernetes cluster (EKS)](https://github.com/medic/medic-infrastructure/blob/master/terraform/aws/dev/eks/access/README.md)
```

> ### Deploy new version
> 

Reviewer: ```suggestion
### Deploy new version

These commands should be run in the `./scripts/deploy` directory in this repo.

```

#### KE
```shell
# Edit tag in users-chis-ke.yaml and then run:

helm upgrade \
      --kube-context arn:aws:eks:eu-west-2:720541322708:cluster/prod-cht-eks \
      --namespace users-chis-prod \
      --values values/users-chis-ke.yaml \

>       users-chis-ke medic/cht-user-management

Reviewer: helm was yelling at me that `Error: repo medic not found` when I ran this. I had to specify a full path to the local repo like this:

```
helm upgrade       --kube-context arn:aws:eks:eu-west-2:7205
41322708:cluster/prod-cht-eks       --namespace users-chis-prod       --values values/users-chis-
ke.yaml       users-chis-ke /home/mrjones/Documents/MedicMobile/helm-charts/charts/cht-user-manag
ement/
```

should we change them all to be an explicit path?

Reviewer: resolved in commit above about path to this chart repo.
